doc/sphinx/docs/python/tutorials/using_population.rst

```diff
@@ -115,3 +115,28 @@ some individual decision vector:
           triggering the fitness function evaluation, but it is also possible to inject
           spurious information into the population (i.e. breaking the relation between
           decision vectors and fitness vectors imposed by the problem)
+
+The best individual in a population can be extracted as:
+
+.. doctest::
+
+    >>> # The decision vector
+    >>> pop1.get_x()[pop1.best_idx()]
```

Comment:
I am wondering, if this is a common enough pattern, shouldn't we have a ``get_best_x()`` and ``get_best_f()`` methods that just call this line?